code-studio/src/js/components/BeeCellEditor.jsx

```diff
@@ -0,0 +1,78 @@
+/* global React */
+
+var BeeCell = require('blockly-mooc/src/maze/beeCell');
```

Comment:
for some reason, it was giving me trouble when I tried that, but I'll mess with it some more and see if I can at least figure out why